### ISSUE TITLE
Crashing 04/01 

### ISSUE THREAD

**Issue by SrMilton** [MarkHC/CSGOSimple#102]
Gaming crashing after last update D: I can inject in lobby but ingame it crashes.

**Comment by Vladislavik36** [comment]
looks like this class is completely wrong, with 0x64 m_fCameraInThirdPerson dont work.

**Comment by anarh1st47** [comment]
Yes, my fail I think, sorry. Agree with @Vladislavik36. Will search solution to fix this

**Comment by anarh1st47** [comment]
@Vladislavik36 (or maybe me) will provide pull request. There is bad char[8] at line 47 on your screen
